CMakeLists.txt

```diff
@@ -224,6 +224,19 @@ IF (NOT Boost_FOUND)
     MESSAGE (SEND_ERROR "[Boost] not found.")
 ENDIF ()
 
+### NRLMSISE-00 Model
+
```

Comment:
so I have explicitly avoided including the nrlmsise model code in an effort to not make it an install-time dependency. Does this undo that?